packages/playground/src/utils/getCapacityUrl.ts

```diff
@@ -0,0 +1,14 @@
+import { NetworkEnv } from "@threefold/grid_client";
+
+const network = process.env.NETWORK || (window as any).env.NETWORK;
```

Comment:
this line isn't needed, it can be passed from the calling function, the file should be only the getCapacityURL function parameterized in the a network var, the usage of that function should be in the right context